src/components/Table/style.module.scss

```diff
@@ -0,0 +1,49 @@
+@import 'assets/styles/mixins';
+@import 'assets/styles/variables';
+
+.table {
+	box-sizing: border-box;
+	width: 100%;
+	height: 686px;
+	background: #f6f6f6;
+	border-radius: 10px;
+	padding: 20px;
+
+	&__header {
+		@include flex($justify: space-between);
+		margin-bottom: 20px;
+	}
+
+	&__actions {
+		@include flex();
+	}
+
+	&__title {
+		font-weight: 500;
+		font-size: 18px;
+		line-height: 22px;
+		color: $second-main-color;
+	}
+}
+
+.button {
```

Comment:
let's change default styles for a mui with a theme for outlined and contained buttons